### PR TITLE
Update Tailscale extension

### PR DESCRIPTION
**Pull request by rosszurowski**
## Description

This commit updates the Tailscale extension with a few notable changes:

- Fixes the "Switch Account" action, which broke in #8287 (sorry, I should have reviewed, but the notification got lost in my GitHub inbox)
- Updates error handling to be a full-screen view, rather than an easy-to-miss, transient toast
- Provides distinct error messages for when the Tailscale CLI path is wrong, when Tailscale isn't even open, and when Tailscale isn't connected to a tailnet
- Provides proper typings from the `tailscale status --json` response

These changes are currently split into two commits, but I'll squash on merging.

## Screencast

https://github.com/raycast/extensions/assets/303731/b561423d-0f87-4293-b385-84de720ca532

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/tailscale/raycast-env.d.ts [hunk 1]
```diff
@@ -40,3 +40,13 @@ declare namespace Arguments {
   /** Arguments passed to the `admin` command */
   export type Admin = {}
 }
+
+
+declare module "swift:*" {
```
Comment (rosszurowski) [inline extensions/tailscale/raycast-env.d.ts:45]: This was automatically added, I assume by the latest version of the Raycast toolkit.

**Comment by rosszurowski** [comment]
I pinged @samlinville for a review, but since the "Switch Accounts" action is broken right now in `main`, I'm going to merge as-is. Happy to make any follow-up changes requested in a separate PR.

File: extensions/tailscale/raycast-env.d.ts (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/tailscale/raycast-env.d.ts:45]: Yes, it's autogenerated and should actually be added to the gitignore file 🙂

**Comment by pernielsentikaer** [comment]
Thanks for your contribution @rosszurowski 🔥 

@samlinville do you have time to check it?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Could you also add a CHANGELOG entry? 🙂

**Comment by rosszurowski** [comment]
Good catch! Added a CHANGELOG entry in the latest update. I suspect Sam won't have time to look until next week, but I'll ask again then.

File: extensions/tailscale/raycast-env.d.ts (hunk 1 above)
Comment (rosszurowski) [reply extensions/tailscale/raycast-env.d.ts:45]: Oh, good to know! I'll include that with this change too.

**Comment by samlinville** [comment]
LGTM, thanks!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**